README.md

```diff
@@ -0,0 +1,492 @@
+
+##### Run unit tests
+
+You can run the whole unit test suite of a model project, using ``pytest``, with ``nox -s tests``.
+This target also prints out the coverage report and save a xml version in ``.ci-reports/``.
+
+##### Run linting
+
+You can run the python linting of the code model project, using ``flake8``, with ``nox -s lint``.
+
+
+### Existing model project
+
+Please note, the maintenance policy of the repository is to consider a model project freezed from the moment when it is intergrated. It means you can open an issue, but the only major ones (bugs breaking the functionality or leading the model to be scientifically unrelevant) will be treated, but no upgrade nor refactoring of the existing model code will be adressed.
+
+If you want to propose a correction to an exixting model, or a new model implementation of an existing model project, please follow the recommendations described in the [Setting-up the environment](#setting-up-the-environment) section to develop in the proper Docker container. If you prefer to develop in a python virtual environment, please use the *Nox* session ``dev_dependencies`` (as described in [Create the proper python development environment](#create-the-proper-python-development-environment)) that will create the proper virtualenv to develop in.
+
+
+### New model project
+
+To start a new model project from scratch, we recommend to adopt the following procedure to develop your contribution. The existing model project are good sources of inspiration of what you will have to do.
+
+You can find in the [Model project files](#model-project-files) section the list of files and directories a model project should includes.
+
+#### Setup the environment
+
+The first step is to initiate the required files that will permits to setup the development environment.
+
+As a reminder of the [Model project files](#model-project-files) section, the required files to define the environment are :
+
+* ``env.yaml`` lists the environment requirements, like the Ubuntu docker base image or the Python version.
+* ``noxfile.py`` is the Nox build tool configuration file that defines all targets available for the model project.
+* ``requirements.txt`` contains all the python dependencies of the project, generated using the ``pip freeze`` command.
+
+##### Create the *env.yaml* file
+
+This file will actually be used later by the github actions workflow in charge of building the docker image of the model project, but we recommend to create and fill it now to fix :
+- the **Ubuntu image** you will work on,
+- the **Python version** you want to use.
+
+The format of this file is the following :
+```
+# This file defines the global environment dependencies for this model project.
+# It is read by the CI/CD workflows in charge of the tests (ai4sim-ci-cd.yaml)
+# and the build of the Docker image (images-ci-cd.yaml) to set the environment.
+python_version: 3.8
```

Comment:
I don't think it will cause conflict, the python version described in the env.yaml file will be installed and used as new default pyhton3. But the "old" one, and its dependencies will no more be used.

In this case (of a pre-configured image), we could document that the python version described in the env.yaml file **must be** the one already available in the docker image to take part of the pre-installed python libs. Is it satisfying enough ?

We could also imagine to make the python version field optional in the env.yaml file, but it will require to manage the case of images without pre-installed python. Maybe it could be adressed later to improve the UX, if needed ?

Reply:
For the moment, additional info in case of images including python is ok for me. Let's discuss later potential improvements.